src/app/InteractionModelEngine.cpp

```diff
@@ -76,9 +77,9 @@ void InteractionModelEngine::Shutdown()
     //
     while (handlerIter)
     {
-        CommandHandlerInterface * next = handlerIter->GetNext();
+        CommandHandlerInterface * pNext = handlerIter->GetNext();
```

Comment:
Given that the declaration is right there next to all the uses of this variable, I'm not sure the `p` prefix is needed... Same for some of the other cases like that in this PR.  And it does make the readability worse...

Reply:
hip::app::InteractionModelEngine::Shutdown()’:
../../src/app/InteractionModelEngine.cpp:80:35: error: declaration of ‘next’ shadows a member of ‘chip::app::InteractionModelEngine’ [-Werror=shadow]
   80 |         CommandHandlerInterface * next = handlerIter->GetNext();
      |                                   ^~~~
In file included from ../../src/transport/SessionManager.h:30,
                 from ../../src/messaging/ExchangeMessageDispatch.h:28,
                 from ../../src/messaging/ApplicationExchangeDispatch.h:28,
                 from ../../src/messaging/ExchangeDelegate.h:27,
                 from ../../src/messaging/ExchangeContext.h:31,
                 from ../../src/app/InteractionModelEngine.h:36,
                 from ../../src/app/InteractionModelEngine.cpp:26:
../../src/credentials/FabricTable.h:388:20: note: shadowed declaration is here
  388 |         Delegate * next = nullptr;
      |                    ^~~~
At global scope:
cc1plus: note: unrecognized command-line option ‘-Wno-unknown-warning-option’ may have been intended to silence earlier diagnostics
cc1plus: all warnings being treated as errors
[2000/4192] ACTION //src/controller/tests:TestWriteChunking_run(//build/toolchain/host:linux_x64_clang)
ninja: build stopped: subcommand failed.
real	1m15.373s

Reply:
renamed it to nextHandler